### PR TITLE
Validate only one AMP runtime version for now

### PR DESCRIPTION
**Pull request by powdercloud**
since only one actually exists.

### CODE REVIEW

**Comment by powdercloud** [comment]
I signed it

File: validator/validator.protoascii [hunk 1]
```diff
@@ -1489,7 +1487,7 @@ tags: {
   attrs: {
     name: "src"
     mandatory: true
-    value_regex: "https://cdn\\.ampproject\\.org/v\\d+\\.js"
+    value: "https://cdn.ampproject.org/v0.js"
```
Comment (cramforce) [inline validator/validator.protoascii]: Also change #L1521 ?

Comment (Gregable) [inline validator/validator.protoascii]: We have a follow up CL for that which splits up the script tag into multiple to address #875.

Comment (cramforce) [inline validator/validator.protoascii]: Cool

**Comment by Gregable** [comment]
@googlebot I signed it!

**Comment by Gregable** [comment]
@googlebot Confirmed

**Pull request merged**